akka-docs/src/test/scala/docs/persistence/PersistenceMultiDocSpec.scala

```diff
@@ -43,11 +47,45 @@ object PersistenceMultiDocSpec {
   //#override-plugins
   trait ActorWithOverridePlugins extends PersistentActor {
     override def persistenceId = "123"
+
     // Absolute path to the journal plugin configuration entry in the `reference.conf`.
     override def journalPluginId = "akka.persistence.chronicle.journal"
+
     // Absolute path to the snapshot store plugin configuration entry in the `reference.conf`.
     override def snapshotPluginId = "akka.persistence.chronicle.snapshot-store"
   }
+
   //#override-plugins
 
+  //#override-config
+
+  //#runtime-config
+  trait ActorWithRuntimeConfig extends PersistentActor {
+    // Variable that is retrieved at runtime, from an external service for instance.
```

Comment:
will it really be used as retrieving from external service? then it must be a `Future[Config]` to avoid blocking, and I guess it will be more complicated

Reply:
The `Config` instance (if provided) must be provided while creating the persistent actor, maybe this comment we wrote is quite misleading since the persistent actor does not care about how this `Config` has been created beforehand.